src/Core/Components/Dialog/Services/DialogOptionsFooterAction.cs

```diff
@@ -14,18 +14,17 @@ public class DialogOptionsFooterAction
     /// <summary />
     public DialogOptionsFooterAction() : this(ButtonAppearance.Default)
     {
-        
     }
 
     /// <summary />
-    internal DialogOptionsFooterAction(ButtonAppearance appearance)
+    internal DialogOptionsFooterAction(ButtonAppearance? appearance)
     {
         Appearance = appearance;
         ShortCut = appearance == ButtonAppearance.Primary ? "Enter" : "Escape";
     }
 
     /// <summary />
-    internal ButtonAppearance Appearance { get; }
+    internal ButtonAppearance? Appearance { get; }
```

Comment:
No need to change this type: that is an **internal** property where the `Appearance` must always be defined.